envFiles/.env.rootless.production

```
@@ -0,0 +1,148 @@
+API_RATE_LIMIT_BUCKET_CAPACITY=500
+API_RATE_LIMIT_REFILL_RATE=100
+
+########## background workers ##########
+
+# Cron schedule for event materialization worker (default: every hour)
+EVENT_GENERATION_CRON_SCHEDULE="0 * * * *"
+
+# Cron schedule for cleanup worker (default: daily at 2 AM UTC)
+CLEANUP_CRON_SCHEDULE="0 2 * * *"
+# https://vitest.dev/config/#watch
+CI=false
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The CI key should go before the CLEANUP_CRON_SCHEDULE key

(UnorderedKey)